fwup.conf

```diff
@@ -105,15 +105,6 @@ file-resource config.txt {
 file-resource cmdline.txt {
     host-path = "${NERVES_SYSTEM}/images/cmdline.txt"
 }
-file-resource u-boot.bin {
```

Comment:
No u-boot anymore?

Reply:
I wasn't building and I removed it since we weren't using any of the functionality. Nerves uses U-Boot environment but not u-boot itself under the hood.